docs/components/DeprecationBanner.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import {Link as GatsbyLink} from 'gatsby'
+import {Note} from '@primer/gatsby-theme-doctocat'
+
+const DeprecationBanner = ({replacementUrl}) => {
+  return (
+    <Note variant="warning" sx={{marginBottom: '40px'}}>
+      <strong>NOTE:</strong> The information on this page is moving to a new home. Please update your links and
+      bookmarks! The new content can be found{' '}
+      <GatsbyLink href={replacementUrl} target="_blank">
+        here
+      </GatsbyLink>
+      .
+    </Note>
```

Comment:
Some suggestions:

- Remove the **Note:** content as I think it's redundant using the Note component
- Updated the text a bit
- If we the link is "internal", do we need a target="_blank"?


```suggestion
    <Note variant="warning" sx={{marginBottom: '40px'}}> 
      This documentation is moving to 
      <GatsbyLink href={replacementUrl}>{replacementUrl}</GatsbyLink> 
      Please update any link or bookmark that points to this page.
    </Note>
```